### ISSUE TITLE
fvconfig load hangs

### ISSUE THREAD

**Issue by jbsbbn** [opennetworkinglab/flowvisor#226]
On a clean install of FV 1.0.4 on Ubnutu 12.04.2 (up to date), if do a fvctl save-config and then fvconfig load, the latter hangs.

**Comment by jbsbbn** [comment]
Here's the exact sequence I did:

```
sudo apt-get update ; sudo apt-get -y dist-upgrade
sudo init 6 && exit

wget http://updates.onlab.us/GPG-KEY-ONLAB
sudo apt-key add GPG-KEY-ONLAB

sudo sh -c 'echo "deb http://updates.onlab.us/debian stable/" >> /etc/apt/sources.list'
sudo apt-get update
sudo apt-get -y install flowvisor

sudo apt-get -y install pwgen
test -f /etc/flowvisor.passwd || sudo sh -c 'pwgen -sB 24 1 > /etc/flowvisor.passwd'
sudo apt-get -y remove pwgen

cat /etc/flowvisor.passwd
sudo -u flowvisor fvconfig generate /etc/flowvisor/config.json

sudo service flowvisor start
```

and then

```
cd
fvctl --passwd-file=/etc/flowvisor.passwd save-config config.json
sudo service flowvisor stop
sed -i -e 's/"api_webserver_port": -1,/"api_webserver_port": 8080,/' config.json
sudo -u flowvisor fvconfig load config.json
```

and that hung.

I tried blowing away the DB and starting over:

```
sudo rm -rf /usr/share/db/flowvisor/FlowVisorDB
cat /etc/flowvisor.passwd
sudo -u flowvisor fvconfig generate /etc/flowvisor/config.json
sudo -u flowvisor fvconfig load config.json
```

That hung too.

Here's the config.json file:

```
{
  "switches": [],
  "flowvisor": [
{
  "api_webserver_port": 8080,
  "db_version": 2,
  "host": "localhost",
  "log_ident": "flowvisor",
  "checkpointing": false,
  "listen_port": 6633,
  "logging": "NOTE",
  "run_topology_server": false,
  "log_facility": "LOG_LOCAL7",
  "version": "flowvisor-1.0.4",
  "config_name": "default",
  "api_jetty_webserver_port": 8081,
  "default_flood_perm": "fvadmin",
  "track_flows": false,
  "stats_desc_hack": false
}
  ],
  "FlowSpaceRule": [],
  "Slice": [
{
  "contact_email": "fvadmin@localhost",
  "admin_status": true,
  "creator": "fvadmin",
  "passwd_salt": "886091067",
  "drop_policy": "exact",
  "config_name": "default",
  "max_flow_rules": -1,
  "name": "fvadmin",
  "controller_port": 0,
  "controller_hostname": "none",
  "flowmap_type": "federated",
  "passwd_crypt": "06d4bcea6dc7f9bef1f7057df2bddbf4",
  "lldp_spam": true
}
  ]
}
```

Lemme know if you need any other info that I can gather.

**Comment by alshabib** [comment]
I forgot to shutdown the DB after loading the configuration... sorry will fix this now.

**Comment by alshabib** [comment]
fixed in 1.0.5 and in cddb28eadd99d6d4558d61b0fea2e0ba8309c436
